domino/routes.py

```diff
@@ -88,12 +88,6 @@ def model_versions_get(self, model_id):
     def model_version_publish(self, model_id):
         return self._build_models_url() + '/' + model_id + '/versions'
```

Comment:
Can you try to have fstring everywhere for string building? I am not sure, if you would want to clean up existing code.